assets/admin/editor-wizard/editor-wizard-modal.js

```diff
@@ -81,10 +91,41 @@ const EditorWizardModal = () => {
 		} );
 	};
 
+	// Choose steps by post type.
+	const stepsByPostType = {
+		course: [ CourseDetailsStep, UpgradeStep, CoursePatternsStep ],
+		lesson: [ LessonDetailsStep, LessonPatternsStep ],
+	};
+	const postType = select( 'core/editor' )?.getCurrentPostType();
+	const steps = stepsByPostType[ postType ];
+
+	// eslint-disable-next-line no-unused-vars
+	const onWizardCompletion = ( wizardData ) => {
+		// TODO Implement actions when wizard is completed
+		closeModal();
+	};
+
+	const updateModalTitle = ( wizardData ) => {
+		if (
+			wizardData.modalTitle !== undefined &&
+			wizardData.modalTitle !== modalTitle
+		) {
+			setModalTitle( wizardData.modalTitle );
+		}
+	};
+
 	return (
 		open && (
-			<Modal onRequestClose={ closeModal } title="I'm a modal!">
-				Code me, please!
+			<Modal
+				className="sensei-editor-wizard-modal"
+				onRequestClose={ closeModal }
+				title={ modalTitle }
+			>
+				<Wizard
+					steps={ steps }
+					onCompletion={ onWizardCompletion }
+					onChange={ updateModalTitle }
```

Comment:
This seems a little confusing to me. The onChange in the `Wizard` and the `updateModalTitle` here seems something more generic. As you said in other comment, I'd remove this and use the title as part of the steps. Otherwise to style it would be a little confusing too since in some steps it's just in a left column, and in the patterns it would be full width.

Reply:
@renatho I have simplified all of this based on input from @fjorgemota here https://github.com/Automattic/sensei/commit/763a9e9ce637a4035adb8eaca5c9d3ecb4fcb128

I would rather let it as it is now and while implementing the first step we can give it a go styling the title inside the steps themselves. What do you think?